reporter-influxdb/src/main/java/io/ultrabrew/metrics/reporters/influxdb/InfluxDBClient.java

```diff
@@ -57,12 +63,29 @@ private CloseableHttpClient getHttpClient() {
   private void doWrite(final String measurement, final String[] tags, final String[] fields,
       final long timestamp)
       throws IOException {
+    if (Strings.isNullOrEmpty(measurement)) {
+      LOGGER.warn("Null or empty measurement.");
+      return;
+    }
+    int rollback = byteBuffer.position();
     byteBuffer.put(measurement.getBytes(UTF_8));
     for (int i = 0; i < tags.length; i += 2) {
+      if (Strings.isNullOrEmpty(tags[i])) {
+        LOGGER.warn("Null or empty tag key in tags array: {} for measurement {}", 
```

Comment:
This has potential to cause a _lot_ of log messages. I'm not sure if we should do some rate limiting or something here (applies to all the logging here of course).

Reply:
This should rarely happen that we have a null tag key or field or value. If we do then it is a user bug so I'm ok flooding their logs with such messages. The one about null tag values would have been potentially much more naggy but since Mika wanted to substitute the string "null" it isn't an issue.